docs/release-promotion/desktop/howto-rc.md

```diff
@@ -1,4 +1,58 @@
-# RC Firefox releases
+# Add a What's New Page to a Firefox RC
+
+## Requirements
+
+* Access to production Balrog
+* A confirmed list of locales that should receive the WNP from Product
+* A confirmed URL of the WNP from Product
+
+## How
+
+* Download the existing Release Blob (for example: Firefox-59.0-build1)
+  * Go to https://aus4-admin.mozilla.org/releases
+  * Search for the release blob you need
+  * Click "Download"
+* Make the following changes to it:
```

Comment:
Sounds like with some small tweaking we could update [create_wnp_blob.py](https://hg.mozilla.org/build/braindump/file/1c9c6cb21a2a/releases-related/create_wnp_blob.py#l15 ) so that it handles some of the manual blob manipulation.

Perhaps we land this for now and file a ticket to update create_wnp_blob.py ?

Reply:
Sure. This is also going to be simplified once I switch automation to submit V9 blobs upfront.